src/applications/check-in/pre-check-in/routes.jsx

```diff
@@ -14,8 +14,9 @@ import ErrorTest from './pages/ErrorTest';
 import { URLS } from '../utils/navigation';
 
 import withFeatureFlip from '../containers/withFeatureFlip';
-import withAuthorization from '../containers/withAuthorization';
-import withForm from '../containers/withForm';
+// import withAuthorization from '../containers/withAuthorization';
+// import withForm from '../containers/withForm';
+import withSession from '../containers/withSession';
```

Comment:
NOTE: so, `withSession` will probably be a good replacement for `withAuth` and `withForm`.